### PR TITLE
Improve data extraction for usabilla agent.

### PR DESCRIPTION
**Pull request by rrodrigu3z**
* Add `data` attribute if present.
* Try to guess `score`.
* Update usabilla_api gem.

### Screenshot

Dry Run:

screen shot 2017-08-21 at 12 28 20 pm

### CODE REVIEW

File: app/models/agents/usabilla_agent.rb [hunk 1]
```diff
@@ -218,18 +218,33 @@ def retrieve_emails
...
         id: r.id,
         custom: r.custom,
-        public_url: r.public_url,
-        button_id: r.button_id,
+        public_url: extract_if_present(:public_url, r),
+        button_id: extract_if_present(:button_id, r),
         created_at: r.date,
-        email: r.email
+        email: extract_if_present(:email, r),
+        data: extract_if_present(:data, r)
```
Comment (duboff) [inline app/models/agents/usabilla_agent.rb:230]: Wouldn't `.data` just generate a ruby object? or is there built in serialization?

Comment (rrodrigu3z) [reply app/models/agents/usabilla_agent.rb:230]: I think there is built in serialization, You can see it in the uploaded screenshot. But let me test with some event propagation.

**Comment by duboff** [comment]
OK in that case LGTM

File: app/models/agents/usabilla_agent.rb (hunk 1 above)
Comment (rrodrigu3z) [reply app/models/agents/usabilla_agent.rb:230]: ok `"{{data.message}}" and {{data.mood}}` works on other agents.

**Pull request merged**